java/rocksjni/write_batch_with_index.cc

```diff
@@ -539,6 +539,23 @@ void Java_org_rocksdb_WBWIRocksIterator_status0(
   rocksdb::RocksDBExceptionJni::ThrowNew(env, s);
 }
 
+/*
+ * Class:     org_rocksdb_WBWIRocksIterator
+ * Method:    refresh0
+ * Signature: (J)V
+ */
+void Java_org_rocksdb_WBWIRocksIterator_refresh0(
+    JNIEnv* env, jobject jobj, jlong handle) {
+  auto* it = reinterpret_cast<rocksdb::Iterator*>(handle);
```

Comment:
Note the cast to rocksdb::Iterator here instead of rocksdb::WBWIIterator, so that we can use the default implementation and return Status::NotSupported.

Reply:
I don't see WBWIIterator to be a child class of Iterator. I don't know how this can be casted.

Reply:
I guess you are right. This seems like a blunder on my part. Thanks for catching this. 
RocksIterator and WBWIRocksIterator inherit from AbstractRocksIterator/RocksInterface on the java side, but they don't have share such an inheritance chain on the C++ side. 

I need to re-think the strategy for WBWIRockIterator ... may be something like just throwing a RocksDBException from Java layer without even going to the C++ layer.

Reply:
Sounds reasonable.